srcpkgs/nvidia/template

```diff
@@ -209,6 +209,8 @@ do_install() {
 
 	# optix ray racing engine
 	vinstall libnvoptix.so.${version} 755 usr/lib
+	ln -sf libnvoptix.so.${version} ${DESTDIR}/${libdir}/libnvoptix.so
+	ln -sf libnvoptix.so.${version} ${DESTDIR}/${libdir}/libnvoptix.so.1
```

Comment:
I don't think `libdir` is set? You should just use `usr/lib` directly.

Reply:
Fixed it, thanks!